sysboundary/ionosphere.cpp

```diff
@@ -38,12 +38,14 @@
 #include "../fieldsolver/ldz_magnetic_field.hpp"
 #include "../common.h"
 #include "../object_wrapper.h"
-#include "vectorclass.h"
-#include "vector3d.h"
 
-#if VECTORCLASS_H >= 200
-#define Vec3d Vec3Dd
-#endif
+#include <Eigen/Dense>
+
+#define Vec3d Eigen::Vector3d
+#define cross_product(av,bv) (av).cross(bv)
+#define dot_product(av,bv) (av).dot(bv)
+#define vector_length(v) (v).norm()
+#define normalize_vector(v) (v).normalized()
```

Comment:
This makes the change minimal in terms of code line diffs, but wouldn't it be better to actually change lines where such calls are used?

Reply:
You are right, of course, in principle.
But I'm waiting with that until all our other boundary-fiddling changes have gone through, to not cause even more conflicts.